ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml

```diff
@@ -11,6 +11,14 @@ base_images:
     name: ocm-2.7
     namespace: edge-infrastructure
     tag: assisted-service
+  assisted-service-index:
+    name: assisted-service-index
+    namespace: edge-infrastructure
+    tag: latest
```

Comment:
for conformity, should we stick with the ``ocm-2.7`` name convention?

Reply:
I kept the naming from what we deploy to quay, at least it stays consistent with its usage in assisted-test-infra: https://github.com/openshift/assisted-test-infra/blob/master/Makefile#L45

But maybe we should change that later?

Reply:
sure